randovania/games/dread/json_data/Artaria.txt

```diff
@@ -735,13 +730,9 @@ Extra - asset_id: collision_camera_006
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Door to Teleport to Dairon (Lower)
-      All of the following:
-          After Artaria - EMMI First Chase
-          Simple IBJ or Use Spin Boost
-  > Door to EMMI Zone Spinner
-      After Artaria - EMMI First Chase and Can Slide
-  > Event - EMMI First Chase
-      Before Artaria - EMMI First Chase
+      Flash Shift or Speed Booster or Simple IBJ or Use Spin Boost
```

Comment:
Does Flash Shift require a walljump to reach this?

Reply:
yeah but seeing as how we're treating beginner wall jumps like this as trivial, i feel flash is enough